### PR TITLE
Generating HTML summary page. Added generate_summary command.

### PR DESCRIPTION
**Pull request by ldrvoderic**

### CODE REVIEW

File: lib/deliver/commands_generator.rb [hunk 1]
```diff
@@ -66,6 +66,17 @@ def run
         end
       end
 
+      command :generate_summary do |c|
+        c.syntax = 'deliver'
```
Comment (KrauseFx) [inline lib/deliver/commands_generator.rb]: That would be `deliver generate_summary`

**Comment by KrauseFx** [comment]
Looks great, thanks for the PR. Have you tested this command locally? I added one little note before it's ready to be merged. Also, please squash the commits after updating the PR and let me know once it's ready :+1:

**Comment by ldrvoderic** [comment]
I did correction for command syntax and also squashed commits. Command is also tested locally and ready for merge. I use generated summary for custom integration with Jenkins CI + Slack confirmation.

**Comment by KrauseFx** [comment]
This is great, thank you so much :+1:

**Pull request merged**